Ecommerce/index.html

```diff
@@ -1,13 +1,32 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>E-commerce</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <!-- Start coding from the lines below for ECOMMERCE TASK -->
-    <!-- to import image from public folder, do it like this = <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+
+<!DOCTYPE html>
+<html><head><title>Task1</title></head>
+<body>

```

Comment:
The list items within the unordered list should be wrapped in 'li' tags individually for proper structure and semantics.